render.go

```diff
@@ -89,6 +89,8 @@ type Options struct {
 	UnEscapeHTML bool
 	// Streams JSON responses instead of marshalling prior to sending. Default is false.
 	StreamingJSON bool
+	// Disables feature that renders http.StatusInternalServerError when an error occurs. Default is false.
```

Comment:
`Disables automatic rendering of http.StatusInternalServerError when an error occurs. Default is false.`